src/components/application_manager/src/display_capabilities_builder.cc

```diff
@@ -157,7 +157,8 @@ bool DisplayCapabilitiesBuilder::IsWaitingForWindowCapabilities(
 void DisplayCapabilitiesBuilder::ResetDisplayCapabilities() {
   SDL_LOG_AUTO_TRACE();
   sync_primitives::AutoLock lock(display_capabilities_lock_);
-  for (auto& window_id : window_ids_to_resume_) {
+  auto window_ids = window_ids_to_resume_;
+  for (auto& window_id : window_ids) {
```

Comment:
```suggestion
  for (const auto& window_id : window_ids) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e